src/wp-includes/class-wp-xmlrpc-server.php

```diff
@@ -335,6 +346,23 @@ public function escape( &$data ) {
 		}
 	}
 
+	/**
+	 * @param IXR_Error|string $error
+	 * @param false $message
+	 */
+	public function error( $error, $message = false ) {
+		// Accepts either an error object or an error code and message
+		if ( $message && ! is_object( $error ) ) {
+			$error = new IXR_Error( $error, $message );
+		}
+
+		if ( function_exists( 'status_header' ) && ! $this->is_enabled ) {
+			status_header( $error->code );
+		}
```

Comment:
Unlike `IXR_Server` this class can't be used outside of WordPress so the function will always exist (see the commit message for 832be877379581e0547e4f3f4a0c0232133801fb on the old PR). 
```suggestion
		if ( ! $this->is_enabled ) {
			status_header( $error->code );
		}
```